Sources/XCTestHTMLReport/Classes/Models/Attachment.swift

```diff
@@ -57,28 +58,22 @@ enum AttachmentName: RawRepresentable {
 
 struct Attachment: HTML
 {
-    var padding = 0
-    var filename: String
-    var path: String
-    var type: AttachmentType?
-    var name: AttachmentName?
+    let padding: Int
+    let filename: String
+    let path: String
+    let type: AttachmentType?
+    let name: AttachmentName?
```

Comment:
Non optionals knowing that a type is forced in the init and the name is set to empty